bindings/rascal/neighbourlist/structure_manager.py

```diff
@@ -71,6 +71,20 @@ def get_dense_feature_matrix(self, calculator):
         return self.managers.get_dense_feature_matrix(
                 calculator._representation)
 
+    def get_sparse_feature_matrix(self, calculator):
```

Comment:
I would agree that "sparse" is not the best naming scheme, as sparse could mean many things, not necessarily what it means here.